code_src/staking/dot/dotArgparserUtil.py

```diff
@@ -0,0 +1,72 @@
+# actions
+# parser arguments
+
+# TODO find a way to pass action as string to add_argument command
+
+def actionSeed(parser_in):
```

Comment:
can you comment above each function in this file what the purpose is somewhat more precisely